feg/cloud/configs/service_registry.yml

```diff
@@ -38,7 +38,7 @@ services:
       csfb:
         port: 9079
       feg_hello:
-        port: 9079
+        port: 9103
```

Comment:
why is it necessary to change the ports?

Reply:
The legacy (non NH) is running on 9079, relay with NH routing capabilities are on 9103, so - you can switch between them by changing the service port mapping.